utils/gridlink_impl.c.src

```diff
@@ -418,7 +419,7 @@ struct cell_pair_DOUBLE * generate_cell_pairs_DOUBLE(struct cellarray_DOUBLE *la
                                                      const DOUBLE max_3D_sep, const DOUBLE max_2D_sep, const DOUBLE max_1D_sep,
                                                      const int enable_min_sep_opt,
                                                      const int autocorr,
-                                                     const int periodic)
+                                                     const int periodic_x, const int periodic_y, const int periodic_z)
 {
     const int64_t nx_ngb = 2*xbin_refine_factor + 1;
     const int64_t ny_ngb = 2*ybin_refine_factor + 1;
```

Comment:
Might be convenient to add a `const int periodic = periodic_x || periodic_y || periodic_z;` and then the condition on line 455 would become at little smaller :)